DOCS-12083-mqtt-update-faq-wording-kt (#1627)
mulesoft/docs-connectors: mqtt3/1.0/modules/ROOT/pages/mqtt3-connector-faq.adoc

==  Does MQTT Connector use an object store for the Enable File Persistence feature?

> The *Enable file persistence* feature is available only on the on-premises installation and uses a file for persistent messages. Note that an object store enables persistent storage for Cloud Hub applications. Refer to the xref:object-store::index.adoc[object store] documentation for details.

Reviewer: ```suggestion
The *Enable file persistence* feature is available only on the on-premises installation and uses a file for persistent messages. Note that an object store enables persistent storage for CloudHub applications. Refer to the xref:object-store::index.adoc[object store] documentation for details.
```

== What are the authentication types supported?

MQTT Connector authentication supports user and password, and certificate-based authentication (TLS or WSS). Refer to the xref:mqtt3-connector-studio-configuration.adoc#configure-authentication[configure credentials for authentication] documentation.

== Does MQTT Connector trace connection and operation executions?

Refer to the xref:mqtt3-connector-troubleshooting.adoc[troubleshoot MQTT Connector] documentation to learn how to enable verbose logging in DEBUG mode for the connector, for Eclipse Paho driver and the Mosquitto broker.

==  What is the server-side configuration required for MQTT Connector to work?

> MQTT Connector works with any MQTT version 3.x compliant server also known as MQTT broker.  To configure the server, refer to your MQTT broker’s documentation,  for example, refer to the https://mosquitto.org/documentation[Eclipse Mosquitto] broker documentation.

Reviewer: ```suggestion
MQTT Connector works with any MQTT version 3.x compliant server also known as MQTT broker. To configure the server, refer to your MQTT broker’s documentation, for example, the https://mosquitto.org/documentation[Eclipse Mosquitto] broker documentation.
```

== Are there any permission, roles, or additional configuration required on the broker side to connect with our credentials using MQTT Connector?

> No, there aren't additional permissions or configuration requirements to connect to MQTT version 3.x compliant MQTT brokers. On the MQTT broker side, you do need to configure the listeners properly.

Reviewer: ```suggestion
No, there aren't any additional permissions or configuration requirements to connect to MQTT version 3.x compliant MQTT brokers. On the MQTT broker side, you do need to configure the listeners properly.
```
